carbon-design-system/carbon
# fix(toggle): don't render focus border when clicked while disabled

Closes #11891 

Prevents the toggle to render a focus border when `:active` while `:disabled`

#### Changelog

**Changed**

- Focus border selector for toggle component

#### Testing / Reviewing

- Click on disabled toggle: should render no focus border
- Click on enabled toggle: should render focus border
